src/reminder/reminder.service.ts

```diff
@@ -61,15 +62,15 @@ export class ReminderService {
       await this.reminderRepository.save(reminder);
     }
     reminder.recurrings = await this.recurringRepository.save(
-      this.getRecursion(reminder.rid, dto.startingDateTime, dto.recursion, dto.customRecursion)
+      this.getRecursionForDb(reminder.rid, dto.startingDateTime, dto.recursion, dto.customRecursion)
```

Comment:
is there need await?

Reply:
no need since `getRecursionForDb` does not return a promise